src/components/ParkDetails/ParkDetails.js

```diff
@@ -6,31 +6,37 @@ import ParkActivities from '../ParkActivities/ParkActivities';
 import React, { useState, useEffect }from 'react';
 import SunriseSunset from '../SunriseSunset/SunriseSunset';
 
+import '../Error/Error.css'
+
 
 const ParkDetails = ({selectedPark, addToFutureTrips}) => {
 
   const [parkDetails, setParkDetails] = useState({});
+  const [error, setError] = useState('')
 
   const getDetails = () => {
     getParkDetailsData(selectedPark.parkCode)
       .then(data => {
         const details = filterParkDetails(data)
         setParkDetails(details)
       })
-  }
+      .catch(err => setError('Something went wrong. Please try again later!'))
```

Comment:
Same question as above, do you catch more specific errors as part of your fetch call?